NOTICE-binary

```diff
@@ -465,3 +465,11 @@ Mustache.js V4.10 - The MIT License (MIT)
 Spur Dashboard V1.1.0 - The MIT License (MIT)
     License Text ( https://github.com/HackerThemes/spur-template/blob/master/LICENSE )
     Copyright 2016 - 2019 Alexander Rechsteiner
+
+Ganglia Core V3.7.1, Ganglia Web V3.7.2 - BSD 3-Clause "New" or "Revised" License
```

Comment:
sorry after thinking about this some more, I think we should remove these since not in the jar and then we can figure out what needs to go into the docker image itself before we release it.